exporter/prometheusexporter/metadata.yaml

```diff
@@ -15,3 +15,8 @@ status:
   - liatrio
   codeowners:
     active: [Aneurysm9]
+
+tests:
+  config:
+    endpoint: "1.2.3.4:1234"
+  skip_lifecycle: true
```

Comment:
is skipping necessary for this exporter? i'd expect that setting the endpoint to `127.0.0.1:<some random port>` would be enough to make this pass no?

Reply:
@Frapschen does this ^^ not work?

Reply:
it works fine